### PR TITLE
Doc: Get started fixes

### PR DESCRIPTION
**Pull request by yannickroger**
I am targeting this branch, because I am fixing bugs so it is backward compatible.

## Changelog

```markdown
### Fixed
Typos in the get started doc
```

## Subject
I did the get started doc and found a few typos. Here are fixes for them.

### CODE REVIEW

File: Resources/doc/getting_started/installation.rst [hunk 1]
```diff
@@ -137,16 +137,27 @@ You can do this by importing them in the routing configuration:
         resource: "@SonataAdminBundle/Resources/config/routing/sonata_admin.xml"
         prefix: /admin
 
-Step 5: Preparing your Environment
+Step 5: Enable the "translator" service
+------------------------------------
+
+The translator service required by SonataAdmin to display all labels properly.
```
Comment (greg0ire) [inline Resources/doc/getting_started/installation.rst]: Your sentence misses a word.

**Comment by yannickroger** [comment]
@greg0ire Fixed

**Comment by greg0ire** [comment]
Thanks!

Can you have a look at the failing build? You have a very simple problem to fix before this can get merged.

**Comment by yannickroger** [comment]
@greg0ire Fixed

**Comment by greg0ire** [comment]
LGTM and by the way, kudos on your commits, they are nice and clean :)

**Comment by OskarStark** [comment]
This is not correct, we support Symfony 2.8...

**Comment by core23** [comment]
Any idea how to write docs for both versions?

**Comment by greg0ire** [comment]
I don't think we should bother about that, people that still use sf < 3 are on old projects, and are seasoned sf developers, they know what `bin/console` means, they can make the translation.

**Comment by OskarStark** [comment]
symfony itself uses app/console instead of bin/console in docs currently

**Comment by greg0ire** [comment]
[Aren't they planning to move to bin/console](https://github.com/symfony/symfony-docs/issues/5898) ?

**Comment by OskarStark** [comment]
> Aren't they planning to move to bin/console ?

yes but we don not fully support symfony 3.0 -> SonataUserBundle

**Comment by ju1ius** [comment]
> Any idea how to write docs for both versions?

Maybe using the composer notation ? Like `symfony-bin-dir/console`, `symfony-var-dir/cache` at al.

```json
// composer.json
{
  "extra": {
    "symfony-app-dir": "app",
    "symfony-web-dir": "web",
    "symfony-var-dir": "var",
    "symfony-bin-dir": "bin",
    "symfony-tests-dir": "tests",
  }
}

**Comment by greg0ire** [comment]
> yes but we don not fully support symfony 3.0 -> SonataUserBundle

You don't have to use 3.0 to be able to use the 3.0 directory structure, and I think will have to switch sometimes anyway

> Maybe using the composer notation ?

Again, not sure it's worth the hassle. By using `bin/console`, you risk less newbies literally typing `symfony-app-dir/console`. Again, the only people that this is important for are newbies, and there are probably more using 3.0 than older versions, don't you think?

**Comment by ju1ius** [comment]
> Again, the only people that this is important for are newbies IMO, and there are probably more using 3.0 than older versions, don't you think?

Agree, using the most up-to-date defaults is certainly more than enough.

**Pull request merged**